src/target/nxpke04.c

```diff
@@ -231,7 +231,7 @@ bool ke04_probe(target_s *t)
 	f->erase = ke04_flash_erase;
 	f->write = ke04_flash_write;
 	f->done = ke04_flash_done;
-	f->erased = 0xFFu;
+	f->erased = 0xffu;
```

Comment:
`U` is the wrong case here

Reply:
According to my script, this was the last instance of lower case unsigned suffix :tada: